package.json

```diff
@@ -1,27 +1,31 @@
 {
-	"name": "ReduxCounterUniversal",
-	"version": "0.0.1",
-	"private": true,
-	"scripts": {
-		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"test": "jest"
-	},
-	"dependencies": {
-		"react": "~15.4.1",
-		"react-native": "0.39.0",
-		"react-redux": "^4.4.6",
-		"redux": "^3.6.0",
-		"redux-thunk": "^2.1.0",
-		"remote-redux-devtools": "^0.5.5"
-	},
-	"devDependencies": {
-		"babel-jest": "17.0.2",
-		"babel-preset-react-native": "1.9.0",
-		"jest": "17.0.3",
-		"jest-react-native": "17.1.0",
-		"react-test-renderer": "~15.4.1"
-	},
-	"jest": {
-		"preset": "react-native"
-	}
+  "name": "ReduxCounterUniversal",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "prop-types": "^15.5.10",
+    "react": "16.0.0-alpha.12",
+    "react-native": "0.48.1",
+    "react-redux": "^5.0.6",
+    "redux": "^3.6.0",
+    "redux-thunk": "^2.1.0",
+    "remote-redux-devtools": "^0.5.5"
+  },
+  "devDependencies": {
+    "babel-jest": "21.0.0",
+    "babel-preset-react-native": "3.0.2",
+    "jest": "21.0.1",
+    "jest-react-native": "18.0.0",
```

Comment:
It seems that `jest-react-native`can be removed.
https://facebook.github.io/jest/docs/en/tutorial-react-native.html#setup

Reply:
Ni hao!

Sure. It is in the doc to do so. And it is simple to test.

In general, it seems a good practice to stick to the react packages. (react-hypescript worked for me but not react-native-hyperscript which was a badly maintained fork).